Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -62,11 +62,7 @@ PaymentMethodsViewModelInputs, PaymentMethodsViewModelOutputs {
     self.showAlert = deletePaymentMethodEventsErrors
       .ignoreValues()
       .map {
-        localizedString(
-          key: "Something_went_wrong_and_we_were_unable_to_remove_your_payment_method_please_try_again",
-          //swiftlint:disable:next line_length
-          defaultValue: "Something went wrong and we were unable to remove your payment method, please try again."
-        )
+        Strings.Something_went_wrong_and_we_were_unable_to_remove_your_payment_method_please_try_again()
```

Comment:
Looks like localized string was available but we didn't use it 🤷‍♂️

Reply:
Actually it was being used. The `localizedString(key:defaultValue:)` function is exactly what `Something_went_wrong...` string function returns. If I'm not wrong, this was implemented that way so we could merge the PR before having the strings translated. But good catch though. 😄